src/ooxml/java/org/apache/poi/openxml4j/opc/OPCPackage.java

```diff
@@ -649,12 +649,11 @@ public PackagePart getPart(PackagePartName partName) {
 	 */
```

Comment:
I see that you started this pull request back in 2016. Have you rebased this to poi trunk (currently progressing towards 3.17 beta 1)?